core/ui/Actions/new-journal.tid

```diff
@@ -0,0 +1,14 @@
+title: $:/core/ui/Actions/new-journal
```

Comment:
This works well, is there a reason we don't do it for the new tiddler action?

Reply:
it's just one line there that would get replaced by another line and I didn't see the benefit of it

Reply:
one line (two counting both the button and the shortcut) would get replaced by other two (or more) lines + the new action tiddler in the new-tiddler case.

I can see the benefit for users simply transcluding action tiddlers - if that's the way you want to go I'm in :sunglasses: ... then we could probably create more action-tiddlers and refactor buttons

Reply:
@Jermolene now we do it for new-tiddler, too